content/group/index.md

```diff
@@ -22,6 +22,14 @@ _raymond at ebi dot ac dot uk_
 
 **Bruhad Dave** is an intern
```

Comment:
Are you going to add a photo?

Reply:
i haven't found a suitable one yet, so not for now